spark/src/main/java/org/apache/iceberg/spark/source/SparkAppenderFactory.java

```diff
@@ -60,6 +61,11 @@
   private StructType eqDeleteSparkType = null;
   private StructType posDeleteSparkType = null;
 
+  // TODO: expose a builder like SparkAppenderFactory.forTable()
```

Comment:
@szehon-ho, I think you had a builder in your PR. Would you be interested to submit separate PR for that once this one is in?

Reply:
Yea sure.